docs/ref/models/constraints.txt

```diff
@@ -203,3 +203,14 @@ For example::
 creates a unique index on ``username`` using ``varchar_pattern_ops``.
 
 ``opclasses`` are ignored for databases besides PostgreSQL.
+
+``db_tablespace``
+-----------------
+
+.. attribute:: UniqueConstraint.db_tablespace
+
+.. versionadded:: 4.0
+
+The name of the :doc:`database tablespace </topics/db/tablespaces>` to use for
```

Comment:
We should update ["Declaring tablespaces for indexes"](https://docs.djangoproject.com/en/3.2/topics/db/tablespaces/#declaring-tablespaces-for-indexes) to the _"Declaring tablespaces for indexes and unique constraints"_.

Reply:
I'll update `tablespace.txt` after https://github.com/django/django/pull/14459 is merged.